Libraries/Network/NetInfo.js

```diff
@@ -196,11 +202,18 @@ const NetInfo = {
     _subscriptions.delete(handler);
   },
 
+  /*
+   * Returns a promise that returns any of the connectivity types listed above.
+   */
   fetch(): Promise {
     return RCTNetInfo.getCurrentConnectivity().then(resp => resp.network_info);
   },
 
   isConnected: {
```

Comment:
If it's not gonna show up on the website, we shouldn't add a comment here, as its obvious by looking at the method signature. We should update Flow types though.